### PR TITLE
Fixed Issue #9679

### PR DESCRIPTION
**Pull request by amit-webkul**
## Issue Reference
#9679 

Added redirection to orders if there are no items in the cart in the reorder section.

### CODE REVIEW

**Comment by ashishkumar-webkul** [comment]
@amit-webkul 

checked and found that the issue has been fixed and working fine. Now when all the items are deleted and then it is redirected to the order page again in case of reorder. Please check the video for reference.

**Video**

https://github.com/bagisto/bagisto/assets/92721837/064ba2bf-bf40-4c5f-bc5b-8e0fd627c601

File: packages/Webkul/Admin/src/Resources/views/sales/orders/create/cart/items.blade.php [hunk 1]
```diff
@@ -395,6 +395,12 @@ class="h-20 w-20 dark:mix-blend-exclusion dark:invert"
                                 }
                             })
                                 .then(response => {
+                                    if (! response.data.data) {
+                                        window.location.reload();;
```
Comment (devansh-webkul) [inline packages/Webkul/Admin/src/Resources/views/sales/orders/create/cart/items.blade.php:399]: Double qoutes.

Comment (amit-webkul) [reply packages/Webkul/Admin/src/Resources/views/sales/orders/create/cart/items.blade.php]: Done

**Comment by ashishkumar-webkul** [comment]
Checked and Found that the issue has been fixed and working fine, Please check the video for reference. 

**Video**

https://github.com/bagisto/bagisto/assets/92721837/84007178-96e7-499b-888e-ce55c0a2a7c8

**Review by jitendra-webkul — APPROVED** [review]

**Pull request merged**